document/src/main/java/com/ritense/document/domain/Document.java

```diff
@@ -45,6 +46,9 @@ public interface Document {
     @JsonProperty
     String internalStatus();
 
+    @JsonProperty
+    Set<CaseTagResponseDto> caseTags();
```

Comment:
Not sure about the name of the method. This might imply to me that you get the domain objects instead of the DTOs. You can tell from the signature however. What do you think?